tools/vp20compiler/main.c

```diff
@@ -499,7 +504,9 @@ void translate(const char* str)
             struct prog_src_register reg = ins.SrcReg[j];
             if (reg.File != PROGRAM_UNDEFINED) {
                 // printf(" - in %d\n", j);
-                assert(!reg.RelAddr); //TODO: A0 rel
+                if(reg.RelAddr) {
```

Comment:
nit: Should be `if (` (note the space)

Reply:
ahh, missed that too, dammit. Thanks for catching it

Reply:
Fixed it for you